homeassistant/components/apple_tv/__init__.py

```diff
@@ -14,12 +11,15 @@
     CONF_PROTOCOL,
     EVENT_HOMEASSISTANT_STOP,
 )
+from homeassistant.core import callback
 from homeassistant.helpers.aiohttp_client import async_get_clientsession
 from homeassistant.helpers.dispatcher import (
     async_dispatcher_connect,
     async_dispatcher_send,
 )
 from homeassistant.helpers.entity import Entity
+from pyatv import connect, exceptions, scan
```

Comment:
3rd party imports should be between standard library imports and homeassistant imports.

Reply:
WTH to be honest. It doesn't matter how I try to run isort, this is how it sorts the imports for me. Do I have to do it in a special way? Just running `isort homeassistant/components/apple_tv/*.py` normally works fine.

Reply:
I created a new venv and installed everything again - it still doesn't work. If I run `isort` on something else in the repo, with imports in the expected order, they don't change. I understand nothing at this point.

```
(havenv) postlund@homer:~/pyatv_dev/home-assistant$ isort --version

                 _                 _
                (_) ___  ___  _ __| |_
                | |/ _/ / _ \/ '__  _/
                | |\__ \/\_\/| |  | |_
                |_|\___/\___/\_/   \_/

      isort your imports, so you don't have to.

                    VERSION 5.5.1

(havenv) postlund@homer:~/pyatv_dev/home-assistant$ isort homeassistant/components/apple_tv/*.py
(havenv) postlund@homer:~/pyatv_dev/home-assistant$ isort homeassistant/components/cast/*.py
(havenv) postlund@homer:~/pyatv_dev/home-assistant$
```

Reply:
Check that you didn't install pyatv from a local repo in the environment. That can trick isort in thinking it's a local application.

Reply:
It's not, it's a brand new venv and I haven't even started Home Assistant so pyatv isn't installed:

```
(havenv) postlund@homer:~/pyatv_dev/home-assistant$ pip uninstall pyatv
WARNING: Skipping pyatv as it is not installed.
```

It's not installed system wide either:

```
(havenv) postlund@homer:~/pyatv_dev/home-assistant$ atvremote
bash: /home/postlund/pyatv_dev/home-assistant/havenv/bin/atvremote: No such file or directory
(havenv) postlund@homer:~/pyatv_dev/home-assistant$
(havenv) postlund@homer:~/pyatv_dev/home-assistant$ deactivate
postlund@homer:~/pyatv_dev/home-assistant$ atvremote
bash: atvremote: command not found
postlund@homer:~/pyatv_dev/home-assistant$
```
Installing pyatv in the venv manually and running isort makes no difference either. Any other ideas?

Reply:
Ok, so... I had a directory named pyatv in the home-assistant directory root which caused this. I guess it confused isort to believe it was installed from a local repo. Removed it and now it worked.